tsl/test/sql/cagg_ddl.sql.in

```diff
@@ -1163,17 +1163,21 @@ SELECT * FROM cashflows;
 -- 3. test named ts
 -- 4. test named bucket width
 -- named origin
+
+\set ON_ERROR_STOP 0
 CREATE MATERIALIZED VIEW cagg_named_origin WITH
 (timescaledb.continuous, timescaledb.materialized_only=false) AS
 SELECT time_bucket('1h', time, 'UTC', origin => '2001-01-03 01:23:45') AS bucket,
 avg(amount) as avg_amount
 FROM transactions GROUP BY 1 WITH NO DATA;
```

Comment:
For unclear [reasons](https://github.com/timescale/timescaledb/pull/6382#discussion_r1543807227), we allowed some `time_bucket` signatures with offset/origin in CAggs (e.g., `time_bucket` with origin and timezone) even we had no actual implementation to support this functionality. 

It looks like we made a mistake when we declared these function signatures in the function cache. Since we decided to block a few `time_bucket` variants known as error-prone (i.e., variable size and offset - which applies here due to the provided timezone) in this PR, we drop the ability to create such CAggs.

Reply:
So this is just a few special cases that are not supported. Could you add a comment above to make that clear?

Reply:
Added a comment in the test file